uwblueprint/jump-math
# query: Student Login Query

## Notion ticket link

Create query for Student Authentication (https://www.notion.so/uwblueprintexecs/Create-query-for-Student-Authentication-60f4de968b3541748668e231975649a9?pvs=4)



## Implementation description
* created TestSessionQueries.ts with query to get test session by access code
* added necessary code to Student Login to ensure that border colour changes depending on code success/failure



## Steps to test
1. Go to http://localhost:3000/student-login
2. Check if border colour is red with invalid access code (ex. 123329)
3. Check if border colour is green with valid access code (ex. 888888)



## What should reviewers focus on?
* right now, GET_TEST_SESSION_BY_ACCESS_CODE returns just id and accessCode since right now I don't need any info (since I am just checking if access code is valid) but this might change in the future - should I change it?
* when you enter the wrong code, border colour is red and error message is displayed --> if you click on any pin after this, the entire pin is reset --> is this okay?


## Checklist
